demo/src/screens/ProductScreen/ProductScreen.js

```diff
@@ -1,13 +1,111 @@
 import React from 'react';
+import {Container, Row, Col, Image, Button} from 'react-bootstrap';
+import {useParams} from 'react-router-dom';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import {Link} from 'react-router-dom';
+import {addOneToCart} from '../../store/StoreHelpers.js';
+
 
 /**
+ * @param {!Object<string,
```

Comment:
This is fine, but in the future, for these especially cumbersome types, you can either typedef or just refer to it as "Object". Thanks!